api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,389 @@ paths:
           description: Partition id
           type: string
           required: true
-        - name: X-Flow-Id
-          in: header
+        - $ref: '#/parameters/XFlowId'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Partition'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a subscription for EventTypes. The subscription is needed to be able to
+        consume events from EventTypes in a high level way when Nakadi stores the offsets and manages the
+        rebalancing of consuming clients.
+        The subscription is identified by its key parameters (owning_application, event_types, consumer_group). If
```

Comment:
Yes, true. PUT makes sense only when making request on specific URL with id/name
I will return it back to POST for now.